parquet/src/encryption/decryption.rs

```diff
@@ -142,16 +142,24 @@ impl CryptoContext {
 /// FileDecryptionProperties hold keys and AAD data required to decrypt a Parquet file.
 #[derive(Debug, Clone, PartialEq)]
 pub struct FileDecryptionProperties {
-    footer_key: Vec<u8>,
-    column_keys: Option<HashMap<Vec<u8>, Vec<u8>>>,
-    aad_prefix: Option<Vec<u8>>,
+    pub footer_key: Vec<u8>,
+    pub column_keys: Option<HashMap<Vec<u8>, Vec<u8>>>,
+    pub aad_prefix: Option<Vec<u8>>,
```

Comment:
These changes may not be strictly necessary. delta-rs has some roundtrip tests between config and properties where I needed to access the contents. Could probably live without.

Reply:
I'll try to get these in, it might not be the best idea security-wise.

Reply:
I think we can skip these for now as I can work around this.